src/pages/home/ReportScreen.js

```diff
@@ -97,6 +101,7 @@ class ReportScreen extends React.Component {
 
         this.state = {
             isLoading: true,
+            isSkeletonViewVisible: true,
```

Comment:
This state variable doesn't seem necessary – could we just use `this.props.isLoadingReportData` instead?

Reply:
While the usage of `this.props.isLoadingReportData` is being discussed, let me clarify why the state is required.

`isLoadingReportData` is being called multiple times, and with load more too. We would ideally want to show the skeletal UI only once. Hence I am adding it to the state and ensuring it only changes once the data is loaded, and not when the user scrolls for more data.

Reply:
Why are we waiting for `isLoadingReportData` _at all_... ?

Reply:
> Why are we waiting for `isLoadingReportData` _at all_... ?

Yeah hence I mentioned, while the usage is being discussed. I am checking alternative solution.